src/screens/AddCash/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
```

Comment:
Basically copied the old component and removed the old code, added in Ratio.

Reply:
You may want to consult Ben if you still should use the iphone X helper library, it could be the thing breaking iPhone 14 Pro.